doc/main.html

```diff
@@ -1622,6 +1655,259 @@ <h3>Complex Example</h3>
+              <li>Second sub item</li>
+            </ol>
+          </li>
+          <li>Second list item</li>
+        </ol>
+      </div>
+
+      <div class="example">
+        <p>
+          Nesting <code>ul</code>:
+        </p> 
+        <pre>
+&lt;ul&gt;
+  &lt;li&gt;First list item
+    &lt;ul&gt;
+      &lt;li&gt;First sub item&lt;/li&gt;
+      &lt;li&gt;Second sub item&lt;/li&gt;
+    &lt;/ul&gt;
+  &lt;/li&gt;
+  &lt;li&gt;Second list item&lt;/li&gt;
+&lt;/ul&gt;</pre>
+        
+        <p>is rendered as:</p>
+        <ul>
+          <li>First list item
+            <ul>
+              <li>First sub item</li>
+              <li>Second sub item</li>
+            </ul>
+          </li>
+          <li>Second list item</li>
+        </ul>
+      </div>
+
+      <div class="example">
+        <p>
+          Nesting <code>ul</code> in <code>ol</code>:
+        </p> 
+        <pre>
+  &lt;ol&gt;
+  &lt;li&gt;First list item
+    &lt;ul&gt;
+      &lt;li&gt;First sub item&lt;/li&gt;
+      &lt;li&gt;Second sub item&lt;/li&gt;
+    &lt;/ul&gt;
+  &lt;/li&gt;
+  &lt;li&gt;Second list item&lt;/li&gt;
+  &lt;/ol&gt;</pre>
+        
+        <p>is rendered as:</p>
+        <ol>
+          <li>First list item
+            <ul>
+              <li>First sub item</li>
+              <li>Second sub item</li>
+            </ul>
+          </li>
+          <li>Second list item</li>
+        </ol>
+      </div>
+
+    </section>
+    <section id="sec-list-complex-example">
+      <h3>Complex List Example</h3>
+
+      <p>
+        The features in <a href="#sec-ordered-lists"></a> and <a href="#sec-list-nesting"></a> may be combined as needed to achieve the required results for a complex list. See below for an example. 
+      </p>
+
+      <div class="example">
+        <p>
+          Nesting <code>ul</code> in <code>ol</code>:
+        </p> 
+        <pre data-include="snippets/list-sample-complex.txt"></pre>
+        
+        <p>is rendered as:</p>
+        <ol>
+          <li>First list item
+            <ul>
+              <li>First sub item
+                <ol type="i">
+                  <li>First sub item's 1st item with small roman numerals</li>
+                  <li>First sub item's 2nd item with small roman numerals</li>
+                </ol>
+              </li>
+              <li>Second sub item
+                <ol type="i" start="3">
+                  <li>Second sub item's 1st item with small roman numerals, starting with "iii" to continue numbers</li>
+                  <li>Second sub item's 2nd item with small roman numerals</li>
+                </ol>
+              </li>
+            </ul>
+          </li>
+          <li>Second list item</li>
+          <li>Third list item</li>
+          <li>Forth list item
+            <ol type="a">
+              <li>Forth list item's a sub item
+                <ul>
+                  <li>1st bullet under sub item
+                    <ul>
+                      <li>1st sub bullet</li>
```

Comment:
We should discourage single items, like we prohibit single sections.

Reply:
Ignoring this as per discussion offline.